Prusa-CW-Firmware/main.cpp

```diff
@@ -72,6 +72,7 @@ enum menu_state {
 #define FW_VERSION  "2.1.4"
 volatile uint16_t *const bootKeyPtr = (volatile uint16_t *)(RAMEND - 1);
 
+const unsigned int ROTATION_START = 200;
```

Comment:
Rename please. All upper-case letters identifiers usually reserved for preprocessor macros.